### PR TITLE
Fix/Disable multiple stylelint errors 

### PR DESCRIPTION
**Pull request by manzoorwanijk**
#### Proposed Changes

See #67467 

- Ignore `unit-allowed-list`
- Fix `selector-pseudo-class-parentheses-space-inside`
- Fix `scss/at-import-no-partial-leading-underscore`
- Fix `property-no-unknown`
- Ignore `no-irregular-whitespace`
- No errors related to `string-quotes` were found
- Fix `no-duplicate-at-import-rules`

#### Testing Instructions

- Do a sanity check on Calypso live link

### CODE REVIEW

File: client/jetpack-cloud/sections/partner-portal/license-preview/style.scss [hunk 1]
```diff
@@ -1,6 +1,5 @@
-@import "@wordpress/base-styles/_breakpoints.scss";
-@import "@wordpress/base-styles/_mixins.scss";
-@import "@wordpress/base-styles/_mixins.scss";
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
```
Comment (manzoorwanijk) [inline client/jetpack-cloud/sections/partner-portal/license-preview/style.scss:2]: It was also a duplicate import here.

File: client/components/pulsing-dot/style.scss [hunk 2]
```diff
@@ -11,7 +11,6 @@
 	border: none;
 	box-shadow: 0 0 0 0 rgba(168, 190, 206, 0.7);
 	border-radius: 100%;
-	v-align: middle;
```
Comment (manzoorwanijk) [inline client/components/pulsing-dot/style.scss:14]: This is an invalid property that has no effect, thus I removed it.

File: client/my-sites/backup/backup-date-picker/style.scss [hunk 3]
```diff
@@ -231,7 +231,7 @@
 
 	// Overriding this rule specifically for backups output. ( the :not selectors add extra specificity )
 	// This is not totally ideal, but a bit better than stringing random selectors together to override specificity.
-	.DayPicker-Day.DayPicker-Day--is-selected:not( .DayPicker-Day--today ):not( .DayPicker-Day--events ) .date-picker__day,
+	.DayPicker-Day.DayPicker-Day--is-selected:not(.DayPicker-Day--today):not(.DayPicker-Day--events) .date-picker__day,
 	// When today is the active day.
```
Comment (manzoorwanijk) [inline client/my-sites/backup/backup-date-picker/style.scss:235]: I am not sure why this is/was not auto-fixed.

File: client/blocks/comments/post-comment-content.scss [hunk 4]
```diff
@@ -1,3 +1,4 @@
+/* stylelint-disable-next-line scss/at-import-no-partial-leading-underscore */
 @import "calypso/assets/stylesheets/shared/_rendered-blocks.scss";
```
Comment (noahtallen) [inline client/blocks/comments/post-comment-content.scss:2]: does this break without the underscore?

File: client/me/security-2fa-enable/style.scss [hunk 5]
```diff
@@ -13,7 +13,6 @@
 }
 
 .security-2fa-enable__app-options {
-	font-color: var(--color-neutral-0);
```
Comment (noahtallen) [inline client/me/security-2fa-enable/style.scss:16]: Why do we need to remove this?

**Review by noahtallen — DISMISSED** [review]
Overall, this looks great, thanks!

The only comment I have is that it probably would have been easier to review if the partial-leading-underscore was in its own PR with some of the miscellaneous changes separated. Not a big deal though, since this was mostly the partial underscore :)

**Comment by noahtallen** [comment]
Comparing `/home` on calypso.live vs wordpress.com, I noticed there's a minor style difference in the "get help" area. Could you verify if that's coming from this PR or unrelated? Otherwise, everything is looking good! :)

https://user-images.githubusercontent.com/6265975/191350766-095a6fcc-1fc1-4ace-b9e5-ccb65e18bd00.mov

File: client/blocks/comments/post-comment-content.scss (hunk 4 above)
Comment (manzoorwanijk) [reply client/blocks/comments/post-comment-content.scss:2]: It breaks the build. May be our build config is not set up properly to use those local paths?

File: client/me/security-2fa-enable/style.scss (hunk 5 above)
Comment (manzoorwanijk) [reply client/me/security-2fa-enable/style.scss:16]: Because of `property-no-unknown` rule. There is no such thing as `font-color` in CSS or is it?

**Comment by manzoorwanijk** [comment]
> Comparing `/home` on calypso.live vs wordpress.com, I noticed there's a minor style difference in the "get help" area. Could you verify if that's coming from this PR or unrelated? Otherwise, everything is looking good! :)
> 

It's not related to this PR, rather it's about async styles that aren't loaded on Calypso Live links. You can see it's the same with any PR.

File: client/me/security-2fa-enable/style.scss (hunk 5 above)
Comment (noahtallen) [reply client/me/security-2fa-enable/style.scss:16]: Oh, duh 🤦 

While I wonder if we should just switch to `color`, it's probably better to maintain the trunk styles, so removing this sounds good to me

**Review by noahtallen — APPROVED** [review]
Thanks for all the fixes!

**Pull request merged**